Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -601,6 +601,11 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 // This property is KVO compliant.
 @property (nonatomic, readonly) _WKWebProcessState _webProcessState WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 
+/*! @constant WKWebViewDataTypeSessionStorage */
+WK_EXTERN NSString * const WKWebViewDataTypeSessionStorage WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionOS(WK_XROS_TBA));
+
+- (void)_fetchDataOfTypes:(NSSet<NSString *> *)dataTypes completionHandler:(WK_SWIFT_UI_ACTOR void (^)(NSData *))completionHandler WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionOS(WK_XROS_TBA));
```

Comment:
I am not sure why this takes `type` as parameters;  the other types are not stored by view so I don't think we expect them to be fetched with this SPI. `_fetchSessionStorageData` seems more reasonable.

Reply:
Oh I see. I was trying to make it generic in case we add more types later. But that makes sense--if the other types aren't even stored on the view, we won't use this SPI for them. Updated!